src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -1268,6 +1268,31 @@ public void canSearchByTitleAndBarcodeWithMissingHoldingsAndItemsAndStillGetInst
       secondInstance.isPresent(), is(true));
   }
 
+  @Test
+  public void canSearchByIsbn1() {
+    canSort("isbn = 0-552-16754-1",      "Interesting Times");
+  }
+
+  @Test
+  public void canSearchByIsbn2() {
+    canSort("isbn = 05-5*",              "Interesting Times");
+  }
+
+  @Test
+  public void canSearchByIsbn3() {
+    canSort("isbn = 9780552167543",      "Interesting Times");
+  }
+
+  @Test
+  public void canSearchByIsbn4() {
+    canSort("isbn = 9780* sortBy title", "Interesting Times", "Temeraire");
+  }
+
+  @Test
+  public void canSearchByIsbn5() {
+    canSort("isbn = 9-7-8-055-2167-543", "Interesting Times");
+  }
```

Comment:
Please can the names of these tests describe the situation being demonstrated, rather than a numeric postfix.

For example, if it is searching by a value without hyphens, maybe it could be called `canSearchForISBNWithoutHyphens`?

What do you think?

Reply:
Fixed.